plugins/assimp/module/vtkF3DAssimpImporter.cxx

```diff
@@ -220,6 +220,7 @@ class vtkF3DAssimpImporter::vtkInternals
 
     vTexture->MipmapOn();
     vTexture->InterpolateOn();
+    vTexture->SetColorModeToDirectScalars();
```

Comment:
I wonder how this can work at all without this line.

Reply:
Direct scalar is the default for 8-bits textures (which is the most common format)

Reply:
ok